contracts/sol6/IKyberStorage.sol

```diff
@@ -88,10 +87,11 @@ interface IKyberStorage {
         view
         returns (bool[] memory entitledRebateArr);
 
-    function getReservesData(bytes32[] calldata reserveIds)
+    function getReservesData(bytes32[] calldata reserveIds, IERC20 src, IERC20 dest)
         external
         view
         returns (
+            bool isListedReserve,
```

Comment:
maybe better 

areAllReservesListed

since check if for all reserves.

Reply:
change as request